text/0000-interactive-app-creation.md

```diff
@@ -0,0 +1,142 @@
+
+If a user were to type in existing flags that are also questions in the interactive workflow (i.e., `ember new my-app --lang en-US --interactive`), it will skip the question entirely. 
+
+When a user types `ember new` into their command line, they will be asked these questions (defaults indicated by a `>`):
+
+- `Is this an app or an addon? (use arrow keys)`
+  - `> app`
+  - `addon`
+- `Please provide the name of your app/addon:`
+- `Please provide the spoken/content language of your app/addon: (use arrow keys)`
+  - `> en-US`
+  - `my computer's default language`
+  - `manually define a different language`
+- `Pick the package manager to use when installing dependencies:  (use arrow keys)`
+  - `> NPM`
+  - `Yarn`
+  - `ignore/skip`
+- `What CI system do you want to use? (use arrow keys)`
+  - `> travis-ci`
+  - `ignore/skip`
+
+#### App/Addon Name
+There will be no default for this question. The user must enter a response. 
+
+#### App/Addon Language
+
+- the default would be `en-US`
+- if your system has a different language set (which can be confirmed by using the `echo $LANG` command in the terminal), then _that_ language would be shown as the second option
+- if your system was already set to the default language, the second option would not be shown. 
+- for "manually define a language", we will validate the response against the allowed language codes
+- we have not added a "skip" option here, because the purpose is to focus on improved accessibility in Ember apps.
+
+#### Package Manager
+
+- ignore/skip has been added to cover the use case where the developer is in a workspace 
+
+#### CI Option
+
+Separate RFCs should further define more options for CI. 
+  
+### Current Limitations
+Explicitly, the goal of this RFC is to only include `ember-cli` flags that already exist. Any other possibilities should be considered future work.
+
+This interactive workflow will also ignore any ~/.ember-cli settings that already exist. A separate RFC should be written that makes it possible for these to things to co-exist and respect the settings of the other.
+
+### Future possibilities
+An example of a future RFC that could be specific to addons, would be to add a `--supported-version` flag for addons and add that to the wizard: 
+
+- `What is the earliest version of Ember that you intend to support? (use arrow keys)`
+  - `> recommended (last two LTS)`
+  - `last LTS`
+  - `manually define a version number`
+
+### Implementation
+This section is not normative, but provided for extra context for folks who might be new to this idea in general. We could use something like [inquirer.js](https://github.com/SBoudrias/Inquirer.js) to implement this wizard, if the mechanism doesn't already exist within the ember-cli codebase. However, it should be explicitly noted that inquirer is only being used as an example of a library that could be used, and this RFC isn't explicitly defining that inquirer.js should be or will be used. 
```

Comment:
I did some exploration for another project for having a CLI wizard, looks like there are other lightweight alternatives like [enquirer](https://github.com/enquirer/enquirer) and [prompts](https://github.com/terkelg/prompts). Let the ember-cli team come up with their own recommended library for the same task.

Reply:
👍 - Absolutely! The idea that is conveyed here is that we will do prompting. Exactly _how_ is an implementation detail in ember-cli (thats why the prose above says "we could use something like Inquirer.js").